topics:
  drone_pose_ned:
    sim: "/ground_truth/ned_frame/drone_pose"
    real: /qualisys/anafi/pose

  helipad_pose_ned:
    sim: "/ground_truth/ned_frame/helipad_pose"
    real: /qualisys/helipad/pose

  drone_pose_helipad_frame:
    sim: "/ground_truth/helipad_frame/drone_pose"
    real: "/ground_truth/helipad_frame/drone_pose"

  helipad_pose_body_frame:
    sim: "/ground_truth/body_frame/helipad_pose"
    real: "/ground_truth/body_frame/helipad_pose"
